tests/integration/conftest.py

```diff
@@ -123,3 +124,282 @@ def sample_video() -> str:
     path_to_video = 'tests/integration/media/cat.mp4'
     assert os.path.exists(path_to_video)
     return path_to_video
+
```

Comment:
All of this hugely bloats `conftest.py`. I'd consider adding a subdirectory `tests/integration/bulk_import/`, and having a dedicated `conftest.py` there, along with the test files for that functionality. Not sure :/